packages/core/hooks/useOpenWithMenuItems/index.tsx

```diff
@@ -8,6 +8,7 @@ import FileFilter from "../../entity/FileFilter";
 import { interaction, metadata } from "../../state";
 
 import styles from "./useOpenWithMenuItems.module.css";
+import AnnotationName from "../../entity/Annotation/AnnotationName";
```

Comment:
tiny nit: up with other entity imports

Reply:
resolved in [55ab0e5](https://github.com/AllenInstitute/biofile-finder/pull/476/commits/55ab0e5082133b9c435eaef5fb5a52e2f1af5720)